build/artifacts.props

```diff
@@ -162,6 +162,7 @@
     <PackageArtifact Include="Microsoft.AspNetCore.SignalR.Protocols.Json" Category="ship" AppMetapackage="true" AllMetapackage="true"/>
     <PackageArtifact Include="Microsoft.AspNetCore.Http.Connections.Common" Category="ship" AppMetapackage="true" AllMetapackage="true"/>
     <PackageArtifact Include="Microsoft.AspNetCore.Http.Connections" Category="ship" AppMetapackage="true" AllMetapackage="true"/>
+    <PackageArtifact Include="Microsoft.AspNetCore.SignalR.Specification.Tests" Category="ship" />
```

Comment:
Move it next to the the packages that aren't in the meta package. Here https://github.com/aspnet/Universe/pull/1202/files#diff-3b8c17af466265008e01f6a7bb8fde76R171